### PR TITLE
Added official kotlin code style to gradle.properties, removed .idea code style configs.

### PR DESCRIPTION
**Pull request by viktorbll**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,5 +1,6 @@
 group=com.epam.drill
 kotlin.native.ignoreDisabledTargets=true
 org.gradle.parallel=true
-#org.gradle.configureondemand=true
-#org.gradle.caching=true
\ No newline at end of file
+#org.gradle.configur eondemand=true
```
Comment (SunFellow) [inline gradle.properties]: Was the space inserted occasionally? Though the line is commented, I don't think it's a good change...

File: gradle.properties [hunk 2]
```diff
@@ -1,5 +1,6 @@
...
+#org.gradle.caching=true
```
Comment (SunFellow) [inline gradle.properties]: Just a side-note: the fact that this line was marked as changed, while nothing has changed here, but the new-line and the new-line is only serving adding the next line is an evidence of necessity to follow the rule: to finish text files with an empty new line.
(Corresponding issue from checkstyle or similar tools: "No new line at the end of file")

File: gradle.properties (hunk 1 above)
Comment (viktorbll) [reply gradle.properties]: It was a typo. Now it's fixed.

Comment (SunFellow) [reply gradle.properties]: The space is present in the final commit after merge.

**Pull request merged**